Maquetacion/scss/tech.scss

```diff
@@ -13,8 +13,13 @@
     justify-content: center;
     margin: 15px 15px;
     text-align: center;
+    transition: 1s;
     width: 220px;
 
+    &:hover {
```

Comment:
Try not to animate properties other than `transform` or `opacity`, otherwise you'll be triggering the whole Painting process of the browser thus lowering the overall performance of the app. :)